maintenance/aws-janitor/account/account.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package account
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/iam"
+)
+
+func GetAccount(sess *session.Session, region string) (string, error) {
+	svc := iam.New(sess, &aws.Config{Region: aws.String(region)})
+	resp, err := svc.GetUser(nil)
+	if err != nil {
+		return "", err
+	}
+	arn, err := parseARN(*resp.User.Arn)
+	if err != nil {
+		return "", err
+	}
+	return arn.account, nil
+}
+
+func parseARN(s string) (*arn, error) {
+	pieces := strings.Split(s, ":")
+	if len(pieces) != 6 || pieces[0] != "arn" || pieces[1] != "aws" {
+		return nil, fmt.Errorf("Invalid AWS ARN: %v", s)
```

Comment:
lowercase errors.

edit: saw `errors.Wrap(err...` later in the diff, possibly we should use the package everywhere.

Reply:
this isn't wrapping an error in this case, `s` is just a string.

Reply:
the `pkg/errors` package has a `errors.Errorf`, which is the equivalent of `fmt.Errorf` plus a stack trace.
so this would be a matter of consistency if others files already use the package instead of the stdlib `fmt / errors` for errors. your call. :)